cpp/ql/test/library-tests/dataflow/taint-tests/stl.h

```diff
@@ -348,7 +348,7 @@ namespace std {
 	class shared_ptr {
 	public:
 		shared_ptr() noexcept;
-		explicit shared_ptr(T*);
+		explicit shared_ptr(T*); shared_ptr(const shared_ptr&) noexcept;
```

Comment:
This is a bit hard to read, is it a pain to put it on its own line?

Reply:
The only pain point is the fact that it'll generate a large diff in the "Files changed" tab on GitHub. But I'll do it, and whoever ends up reviewing this can view the changes commit-by-commit to avoid inspecting the inflated diff.

Reply:
Fixed in 32a8b9a857eb66a5b471de771dff380a61654e7b.